clang/test/CodeGen/debug-info-file-checksum.c

```diff
@@ -15,4 +15,4 @@
 // Ensure #line directives without name do emit checksums
 // RUN: %clang -emit-llvm -S -g -gcodeview -x c %S/Inputs/debug-info-file-checksum-line.cpp -o - | FileCheck %s --check-prefix CHECKSUM
 
-// CHECKSUM: !DIFile(filename: "{{.*}}debug-info-file-checksum-line.cpp", directory:{{.*}}, checksumkind: CSK_MD5, checksum: "7b568574d0e3c56c28e5e0234d1f4a06")
+// CHECKSUM: !DIFile(filename: "{{.*}}debug-info-file-checksum-line.cpp", directory:{{.*}}, checksumkind: CSK_MD5, checksum: "4456fffb9436b349cea43c698a726b08")
```

Comment:
This change doesn't look right, why did checksum change here? I don't see changes to `debug-info-file-checksum-line.cpp` file and therefore I don't expect checksum to be different. Am I missing something?